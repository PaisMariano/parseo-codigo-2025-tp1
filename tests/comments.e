class COMMENTS
feature
    greet
        do
            -- este es un comentario
            print ("Hi!%N") -- comentario inline
        end
end
