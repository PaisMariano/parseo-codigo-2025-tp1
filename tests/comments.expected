TOKEN_CLASS -> class
TOKEN_IDENTIFIER -> COMMENTS
TOKEN_FEATURE -> feature
TOKEN_IDENTIFIER -> greet
TOKEN_DO -> do
TOKEN_IDENTIFIER -> print
TOKEN_LPAREN -> (
TOKEN_STRING -> "Hi!%N"
TOKEN_RPAREN -> )
TOKEN_END -> end
TOKEN_END -> end
